package.py

```diff
@@ -0,0 +1,27 @@
+name = "aTools"
+
+version = "2.03"
+
+authors = [
+    "Alan Camilo",
+    "Michael Klimenko",
+    "Jeremy Andriambolisoa",
+]
+
+description = \
+    """
+    aTools is the ultimate free utilities belt for Maya animators
+    """
+
+
+requires = [
+    "python-3+",
+    "maya-2025"
```

Comment:
Same as Red9 PR, version can be maya-2025+ since all the future versions of Maya will be Python 3 compatible